src/TaskRunner/Commands/ComponentCheckCommands.php

```diff
@@ -586,4 +592,9 @@ private function testPackages()
         ];
     }
 
+    private function printRecommendedWarningMessage(): string
```

Comment:
Hi @crgomes ,
Can you add here a docblock?
Btw, need to check why was not catch by phpcs.